export class DataCloneErrorFactory {
    create() {
        try {
            return new DOMException('', 'DataCloneError');
        }
        catch (err) {
            const exception = new Error();
            exception.code = 25;
            exception.name = 'DataCloneError';
            return exception;
        }
    }
}
//# sourceMappingURL=data-clone-error.js.map